docs/events/projections/healthchecks.md

```diff
@@ -0,0 +1,13 @@
+# Async Daemon HealthChecks
+
+Marten supports a customizable [HealthChecks](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-7.0). This can be useful when running the async daemon in a containerized environment such as Kubernetes. The check will verify that no projection's progression lags more than `maxEventLag` behind the `HighWaterMark`. The default `maxEventLag` is 100.
```

Comment:
**SUGGESTION:** It'd be good to briefly explain what's max event lag.

Thank you for adding documentation :+1: